config.ts

```diff
@@ -0,0 +1,8 @@
+export interface Config {
```

Comment:
Document the fields. Probably just along the lines of https://code.googlesource.com/gocloud/+/master/profiler/profiler.go.